tests/conftest.py

```diff
@@ -4618,3 +4619,67 @@ def finalizer():
 
     request.addfinalizer(finalizer)
     return factory
+
+
+@pytest.fixture(scope="session", autouse=True)
+def patch_consumer_toolbox_with_secret():
+    """
+    Patch the rook-ceph-tools deployment with ceph.admin key. Applicable for MS platform only to enable rook-ceph-tools
+    to run ceph commands until we have the fix for rook-ceph-tools in consumer cluster
+
+    """
+    current_cluster = config.cluster_ctx
+
+    # Get the secret from provider if MS multicluster run
+    if (
+        config.multicluster
+        and config.ENV_DATA.get("platform", "").lower()
+        in constants.MANAGED_SERVICE_PLATFORMS
+    ):
```

Comment:
```suggestion
    if not (
        config.multicluster
        and config.ENV_DATA.get("platform", "").lower()
        in constants.MANAGED_SERVICE_PLATFORMS
    ):
        return
```

You can save the one indention in the whole block below.

Also as it's session scope fixture and in the case it's the deployment there will not be any OCP/ODF installed. In such case you should skip this code as well.

Reply:
Made the changes